style.css

```diff
@@ -0,0 +1,84 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  background-image:
+    url(images/ellipse_3.png),
+    url(images/Ellipse_4.png),
+    url(images/Ellipse_6.png),
+    url(images/ellipse_11.png),
+    url(images/Rectangle_4.png),
+    url(images/Rectangle_6.png),
+    url(images/Rectangle_5.png),
+    url(images/Rectangle_7.png),
+    url(images/ellipse_12.png),
+    url(images/ellipse_5.png),
+    url(images/ellipse_7.png),
+    url(images/Ellipse_13.png);
+  background-position-x: 128%, 57%, 6%, -6%, 104%, -11%, 50%, 27%, 71%, 30%, 65%, 68%;
+  background-position-y: 22%, 87%, 95%, 20%, 16%, 18%, 90%, 20%, 70%, 21%, 16%, 72%;
+  background-repeat: no-repeat;
```

Comment:
- [ ] As suggested by the previous code reviewer, I think it would be better to adjust your background image to match that of the Figma design. 
- [ ] I think the root of the problem is to export all those little shapes in the Figma design individually and then try to fix them in specific places on your page. what would be better is instead clicking on the `header-shape mobile` option on the left-side toolbar on the Figma design, and after clicking it, you can export it from the right-side toolbar. That one background image you can use it as a `background image` in your design.
or you can click on the picture in the above-mentioned comment, download it and use it in your design.